docs/quickstart.rst

```diff
@@ -26,37 +24,18 @@ Ensure aws credentials are configured under the user running aws_ir as documente
 Setup Roles with Cloudformation
 *************************************
 
-A cloudformation stack has been provided to setup a group and a responder role.  *Note that this roles has a constraint
-that all your responders use MFA.*
+A cloudformation stack has been provided to setup a group and a responder role.
```

Comment:
Does this stack still not have an MFA required constraint for role assumption?

Reply:
Self resolved.  I see it below.

Reply:
yeah - I just moved the doc stuff around. Trying to get "quickstart" leaner